packages/ignite/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ignite",
+  "name": "react-native-ignite",
```

Comment:
Yes, there's also a devDependency located in the `ignite-integration-testing` `package.json` too.